index.html

```diff
@@ -0,0 +1,185 @@
+        </div>
+      </section>
+
+      <section id="Program" class="program_section">
+        <h2>Main Program</h2>
+        <ul class="program">
+          <li class="program_1 d-flex flex-center">
+            <div>
+              <img
+                src="https://cdn-icons-png.flaticon.com/512/6947/6947394.png"
+                alt="welcome"
+              />
+            </div>
+            <h3>welcome</h3>
+            <p>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nulla
+              quidem possimus vitae laborum non aliquid molestiae.
+            </p>
+          </li>
+          <li class="program_2 d-flex flex-center">
+            <div>
+              <img
+                src="https://cdn-icons-png.flaticon.com/512/6947/6947384.png"
+                alt="park"
+              />
+            </div>
+            <h3>About Park</h3>
+            <p>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nulla
+              quidem possimus vitae laborum non aliquid molestiae.
+            </p>
+          </li>
+          <li class="program_3 d-flex flex-center">
+            <div>
+              <img
+                src="https://img.icons8.com/external-xnimrodx-lineal-color-xnimrodx/344/external-website-seo-xnimrodx-lineal-color-xnimrodx-4.png"
+                alt="quality"
+              />
+            </div>
+            <h3>Gorilla naming</h3>
+            <p>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nulla
+              quidem possimus vitae laborum non aliquid molestiae.
+            </p>
+          </li>
+          <li class="program_4 d-flex flex-center">
+            <div>
+              <img
+                src="https://cdn-icons-png.flaticon.com/512/7305/7305574.png"
+                alt="gorilla"
+              />
+            </div>
+            <h3>Give family</h3>
+            <p>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nulla
+              quidem possimus vitae laborum non aliquid molestiae.
+            </p>
+          </li>
+          <li class="program_5 d-flex flex-center">
+            <div>
+              <img
+                src="https://cdn-icons-png.flaticon.com/512/6947/6947410.png"
+                alt="family"
+              />
+            </div>
+            <h3>Closure</h3>
+            <p>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nulla
+              quidem possimus vitae laborum non aliquid molestiae.
+            </p>
+          </li>
+        </ul>
+        <div class="btn-program">
+          <button type="button">Donate to gorillafamilies</button>
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+        <div class=""></div>
+      </section>
+
+      <section id="Team" class="featured_section">
+        <h2>featured Speakers</h2>
+        <ul class="teams"></ul>
+        <div class="see-more">
+          <button type="submit">
```

Comment:
- [ ] Please kindly make sure that the `More` button for the speakers section does not appear in the desktop screen. Instead please ensure that it has six individual articles, each with an image, title, subtitle, and description as specified in the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164577956-835fc82a-4c33-43bc-b5a0-feb81a531f60.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164578005-44f4772b-f53f-4dab-a15b-90ae107bd3e4.png)

Reply:
resolved 👍 but the `More` button should be present in the mobile view. It would be great if you could fix that.